app/src/main/java/org/mozilla/focus/fragment/UrlInputFragment.kt

```diff
@@ -244,8 +245,12 @@ class UrlInputFragment :
 
         context?.let {
             if (!Settings.getInstance(it).shouldShowFirstrun()) {
+
                 // Only show keyboard if we are not displaying the first run tour on top.
                 showKeyboard()
+
+                @Suppress("MaxLineLength")
+                toolbarBackgroundView.background = ContextCompat.getDrawable(context!!, R.drawable.animated_background_url)
```

Comment:
Please just use a line break instead of suppressing this here 😅

Reply:
Done.